libdevice/cmath_wrapper_fp64.cpp

```diff
@@ -25,6 +25,10 @@ double ceil(double x) { return __devicelib_ceil(x); }
 DEVICE_EXTERN_C_INLINE
 double copysign(double x, double y) { return __devicelib_copysign(x, y); }
 
+extern "C" SYCL_EXTERNAL double __devicelib_scalbln(double, long);
```

Comment:
Is this line required? I think it's not, since this function is declared in `device_math.h`, which is included by this file.